docs/pages/_meta.js

```diff
@@ -29,6 +29,7 @@ export default {
   },
   guides: "Guides",
   templates: "Templates",
+  dictionary: "Solidity to MUD Dictionary",
```

Comment:
What do you think of calling this "MUD for Solidity programmers"? (leaning on naming of books like [Rust for Embedded C Programmers](https://opentitan.org/book/doc/rust_for_c_devs.html))

Reply:
Done, good idea.